docs/source/clusters.rst

```diff
@@ -636,17 +638,19 @@ Example: S3 Events Cluster
       "region": "us-east-1"
     }
 
-This configures the two buckets to notify the classifier function in this cluster when new objects
-arrive in the bucket, and authorizes the classifier to download objects from either bucket.
+This configures the two buckets (``bucket_name_01`` and ``bucket_name_02``) to notify the classifier
+function in this cluster when new objects arrive in the bucket at the specified (optional) prefix(es),
+provided the objects have the specified (optional) suffix(es). Additionally, this will authorize the
```

Comment:
Is it `AND` or `OR` logic if we have both prefix(es) and suffix(es) configured for a bucket?

Reply:
I think it's `and`.. but they're both optional

https://www.terraform.io/docs/providers/aws/r/s3_bucket_notification.html

Reply:
Aha, you are right. It will be `AND` logic. There is an example in aws doc page https://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html.